keras_eval/metrics.py

```diff
@@ -332,3 +333,44 @@ def get_top1_entropy_stats(y_probs, labels, entropy, verbose=0):
     n_errors = np.array(n_errors)
 
     return correct_list, errors_list, n_correct, n_errors
+
+
+def get_top_k_sensitivity(concepts, y_true, y_probs, top_k, round_decimals=7):
```

Comment:
Why is this computed outside the `get_metrics()` function?
Half of the code is the same isn't it?

Reply:
I created a new function to break down the `get_metrics` function a bit, but I agree that follows the same structure. I removed `get_top_k_sensitivity ` and added the code back to `get_metrics`.